docs/simple_switch.md

```diff
@@ -36,6 +122,7 @@ header_type intrinsic_metadata_t {
         egress_rid : 16;
         resubmit_flag : 8;
         recirculate_flag : 8;
+        clone_spec : 32;   // TBD - should this be added in order for P4_14 clone to work?
```

Comment:
`clone_spec` must come from standard metadata, `intrinsic_metadata.clone_spec` is not used anywhere AFAIK

Reply:
I removed this line.